Tools/scripts/apj_tool.py

```diff
@@ -7,6 +7,24 @@
 
 import argparse
 
+def to_ascii(s):
+    '''get ascii string'''
+    if sys.version_info.major >= 3:
+        return str(s, 'ascii')
+    else:
+        return str(s)
+
+def to_bytes(s):
+    '''get bytes string'''
+    if sys.version_info.major >= 3:
+        if isinstance(s,bytes):
+            return s
+        if not isinstance(s,str):
```

Comment:
```suggestion
        if isinstance(s,int):
```

I think that's the only thing you can pass into `chr(...)`

Reply:
fixed